src/components/breadcrumbs/breadcrumbs.types.ts

```diff
@@ -0,0 +1,28 @@
+import { Link } from '@limetech/lime-elements';
+
+export interface BreadcrumbItem {
+    /**
+     * label displayed on the step.
+     */
+    text: string;
+
+    /**
+     * Icon of the step.
+     */
+    icon?: string;
+
+    /**
+     * If `true`, text will be rendered as a tooltip
+     */
+    iconOnly?: boolean;
+
+    /**
+     * If supplied, the breadcrumbs steps will be a clickable links.
+     */
+    link?: Omit<Link, 'target' & 'text'>;
```

Comment:
not sure if this is the right way to do it, and not sure if we need to add something to the docs above. For example, write something like 
> only `title` and `href` are allowed in the link interface

?

Reply:
I'm not sure how it helps, since no errors or warnings are thrown anywhere, if I add `target` in an example, like below

<img width="448" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/c61b3eac-c350-40e4-ad82-607abc41366a">

Reply:
There is no type for the `items` array in your example, so how can it complain? You are allowing anything :smile: 
Try this
```ts
class BreadcrumbsExample {
    private items: BreadcrumbItem[] = [...]
}
```